File: src/src/EjerciciosDeCondicionalesMartes.java
public class EjerciciosDeCondicionalesMartes {
    public static void main(String[] args) throws Exception {

        System.out.println("inicio ejercicio 1");
        /*
         * Realiza un programa que reciba dos números por teclado e indique cuál es
         * mayor o si son iguales.
         */
        int num1 = 31;
        int num2 = 31;
        if (num1 > num2) {
            System.out.println("el número mayor es el " + num1);
        } else if (num1 < num2) {
            System.out.println("el número mayor es el " + num2);
        } else {
            System.out.println("los dos números son iguales");
        }
        System.out.println("   fin ejercicio 1");

        System.out.println("inicio ejercicio 2");
        /*
         * Crea un programa que pida al usuario dos números y muestre el resultado de su
         * división. Si el segundo número es 0, debe mostrar un mensaje de error.
         */
        double numa = 10;
        double numb = 3;
        if (numb == 0) {
            System.out.println("el segundo número es cero");
        } else {
            double div = numa / numb;
            System.out.println("el resultado de la división es " + div);

        }
        System.out.println("   fin ejercicio 2");

        System.out.println("inicio ejercicio 3");
        /*
         * Realiza un programa que clasifique un triángulo tras recibir el tamaño de sus lados. Se debe clasificar como triángulo rectángulo, isósceles, equilátero o escaleno.
         */
        int a=3;
        int b=4;
        int c=8;
        /* se pueden comparar las tres a la vez así if (a==b==c){*/
        if ((a+b<=c)||(a+c<=b)||(b+c<=a)){
            System.out.println("No se puede hacer un triángulo con las medidas que has proporcionado");
        }else if (a==b && b==c){
            System.out.println("Triángulo Equilatero");
        }else if (a!=b && a!=c && b!=c){
            System.out.println("Triángulo Escaleno");
        }else if (a==b || a==c || b==c){
            System.out.println("Triángulo Isósceles");
        }else if (Math.pow(a, 2) + Math.pow(b, 2) == Math.pow(c, 2) ||
                  Math.pow(a, 2) + Math.pow(c, 2) == Math.pow(b, 2) || 
                  Math.pow(b, 2) + Math.pow(c, 2) == Math.pow(a, 2)){
            System.out.println("Triángulo Rectángulo");
        }else{
            System.out.println("Los datos introducidos no son válidos");
        }
        System.out.println("   fin ejercicio 3");

        System.out.println("inicio ejercicio 4");
        /* Escribe un programa que reciba un año y nos diga si es bisiesto o no. */
        int anio=1987;
        if (anio%4==0){
            System.out.println("El año es bisiesto");
        }else{
            System.out.println("El año no es bisiesto");
        }
        System.out.println("   fin ejercicio 4");

        System.out.println("inicio ejercicio 5");
        /*
         * Realiza un programa que pida el día de la semana (del 1 al 7) y escriba el día correspondiente. Si introducimos otro número nos da un error.
         */
        int dia=8;
        switch (dia) {
            case 1:
                System.out.println("Es lunes");                
                break;
            case 2:
                System.out.println("Es martes");
                break;
            case 3:
                System.out.println("Es miércoles");
                break;
            case 4:
                System.out.println("Es jueves");
                break;
            case 5:
                System.out.println("Es viernes");
                break;
            case 6:
                System.out.println("Es sábado");
                break;
            case 7:
                System.out.println("Es domingo");
                break;
            default:
                System.err.println("El número introducido no esta entre el 1 y el 7");
                break;
        }
        System.out.println("   fin ejercicio 5");

        System.out.println("inicio ejercicio 6");
        /*
         * Realiza un programa que pida un número entero entre uno y doce e imprima el número de días que tiene el mes correspondiente.
         */
        int mes=13;

        /* con el switch  se podria poner case 1,3,5,7,8,10,12; */
        if (mes==2){
            System.out.println("Tiene 28 días");
        }else if (mes==1 || mes==3 || mes==5 || mes ==7 || mes==8 ||mes==10 ||mes==12){
            System.out.println("Tiene 31 días");
        }else if (mes==2 || mes==4 || mes==6 || mes==9 || mes==11){
            System.out.println("Tiene 30 días");
        }else{
            System.out.println("no has indicado un número entre 1 y 12");
        }

        System.out.println("   fin ejercicio 6");

        System.out.println("inicio ejercicio 7");
        /*
         * Dado3númerosenteros, mostrar en consola los números de forma ascendente, de
         * menor a mayor. Ejemplo Números = 9, 50, 4 Salida = 4, 9, 50.
         */
        
        int aa=7;
        int bb=5;
        int cc=3;
        int mayor, medio, menor;
        if (aa>=bb && aa>=cc){
            mayor=aa;
            if (bb>=cc){
                medio=bb;
                menor=cc;
            }else{
                medio=cc;
                menor=bb;

            }
        }else if(bb>=aa && bb>=cc){
            mayor=bb;
            if (aa>=cc){
                medio=aa;
                menor=cc;
            }else{
                medio=cc;
                menor=aa;
            }
        }else{
            mayor=cc;
            if (bb>=aa){
                medio=bb;
                menor=aa;
            }else{
                medio=aa;
                menor=bb;
            }
        }
        System.out.println("Lo números ordenados son :"+menor+","+medio+","+mayor);




        System.out.println("   fin ejercicio 7");

        System.out.println("inicio ejercicio 8");
        /*
         * Dadounnúmeroentero imprimir en consola el valor correspondiente en la tabla del 9. Ejemplo Entrada = 3 Salida = 27.
         */
        int numdado=10;
        int resultado=numdado*9;
        System.err.println(resultado);
        System.out.println("   fin ejercicio 8");

        System.out.println("inicio ejercicio 9");
        /*
         * Realizar el ejercicio en el que se ingrese 4 notas del alumno, mostrar el promedio de las 4 notas e indicar si está APROBADO caso contrario NO APROBADO.
         */
        double nota1=5.3;
        double nota2=5.5;
        double nota3=8;
        double nota4=7;
        double medianota=(nota1+nota2+nota3+nota4)/4;
        System.err.println("La nota media es :"+medianota);
        if (medianota>=5){
            System.out.println("Aprobado");
        }else{
            System.out.println("Suspendido");
        }
        System.out.println("   fin ejercicio 9");

        System.out.println("inicio ejercicio ");
        System.out.println("   fin ejercicio ");

        System.out.println("inicio ejercicio ");
        System.out.println("   fin ejercicio ");
    }
   
}

File: bin/src/EjerciciosDeCondicionalesMartes.py
print("inicio ejercicio 1")
num1=31
num2=31
if (num1>num2):
    print("el número mayor es el "+ str(num1))
elif (num2<num2):
    print("el número mayor es el "+ str(num2))
else:
    print("los dos números son iguales")
print("   fin ejercicio 1")

print("inicio ejercicio 2")
numa=10
numb=3
if (numb==0):
    print("el segundo número es cero")  
else:
    div=numa/numb
    print("el resultado de la división es "+ str(div))                                       

print("   fin ejercicio 2")

print("inicio ejercicio 3")
a=1
b=2
c=4
if (a+b<=c) or (a+c<=b) or (b+c<=a):
    print("No se puede hacer un triángulo con las medidas que has proporcionado")
elif (a==b==c):
    print("Triángulo Equilatero")
elif (a!=b and a!=c and b!=c):
    print("Triángulo Escaleno")
elif (a==b or a==c or b==c):
    print("Triángulo Isósceles")
elif (a**2 + b**2 == c**2) or (a**2 + c**2 == b**2) or (b**2 + c**2 == a**2):
    print("Triángulo Rectángulo")
else:
    print("Los datos introducidos no son válidos")


print("   fin ejercicio 3")

print("inicio ejercicio 4")
anio=1987
if anio%4==0:
    print("El año es bisiesto")
else:
    print("El año no es bisiesto")

print("   fin ejercicio 4")

print("inicio ejercicio 5")
dia=8
if dia==1:
    print("Es lunes")
elif dia==2:
    print("Es martes")
elif dia==3:
    print("Es miércoles")
elif dia==4:
    print("Es jueves")
elif dia==5:
    print("El viernes")
elif dia==6:
    print("Es sábado")
elif dia==7:
    print("Es domingo")
else:
    print("El número introducido no esta entre el 1 y el 7")

print("   fin ejercicio 5")

print("inicio ejercicio 6")
mes=13
if mes==2:
    print("Tiene 28 días")
elif mes==1 or mes==3 or mes==5 or mes==7 or mes==8 or mes==10 or mes==12:
    print("Tiene 31 días")
elif mes==2 or mes==4 or mes==6 or mes==9 or mes==11:
    print("Tiene 30 días")
else:
    print("no has indicado un número entre 1 y 12")


print("   fin ejercicio 6")

print("inicio ejercicio 7")
aa=7
bb=5
cc=3
"""
num1 =9
num2 =50
num3= 8
if num1>num2:
    num1,num2=num2,num1
if num2>num3:
    num2,num3=num3,num2
if num1>num2:
    num1,num2=num2,num1    
    print("EL ORDEN ES :{NUM1}, {NUM2},{NUM3})
"""
if (aa>=bb and aa>=cc):
    mayor=aa
    if (bb>=cc):
        medio=bb
        menor=cc
    else:
        medio=cc
        menor=bb            
elif(bb>=aa and bb>=cc):
    mayor=bb
    if (aa>=cc):
        medio=aa
        menor=cc
    else:
        medio=cc
        menor=aa            
else:
    mayor=cc
    if (bb>=aa):
        medio=bb
        menor=aa
    else:
        medio=aa
        menor=bb
print("Lo números ordenados son :"+ str(menor)+","+str(medio)+","+str(mayor))


print("   fin ejercicio 7")

print("inicio ejercicio 8")
numdado=10
resultado=numdado*9
print(resultado)
print("   fin ejercicio 8")

print("inicio ejercicio 9")
nota1=5.3
nota2=5.5
nota3=8
nota4=7
medianota=(nota1+nota2+nota3+nota4)/4
print("la nota media es :"+str(medianota))
if (medianota>=5):
    print("Aprobado")
else:
    print("Suspendido")

print("   fin ejercicio 9")

print("inicio ejercicio ")
print("   fin ejercicio ")

print("inicio ejercicio ")
print("   fin ejercicio ")

print("inicio ejercicio ")
print("   fin ejercicio ")